test_login.py

```diff
@@ -0,0 +1,25 @@
+import requests
+
+base_url = 'https://python-qa-training.herokuapp.com/api/auth/login'
+email = 'sievdokymov@lohika.com'
+password = '12345678'
+
+def login(email,password):
+    res = requests.post(base_url, json={"email":email, "password":password})
```

Comment:
According to PEP-8, it misses whitespace between json "email": and the email var (same for password).

More about PEP-8 - https://www.python.org/dev/peps/pep-0008/

Reply:
@bochkala fixed in latest version submitted as "homework1-5" pull request - appropriate code style changes were made